test-files/cmd/slow/ConflateCmdStatsAllDataTypesTest.sh.stdout

```diff
@@ -1,3 +1,9 @@
+21:07:57.398 INFO   ...ore/io/OsmMapReaderFactory.cpp( 169) Loading map from ...d/slow/ConflateCmdStatsTest/all-data-types-out.osm...
```

Comment:
Almost all of the stdout changes in these tests are directly related to new stats generated by the new matchers.